### ISSUE TITLE
C2V cannot translate wayland-egl-backend.h

### ISSUE THREAD

**Issue by Shinyzenith** [vlang/c2v#48]
Error:
```sh
C to V translator 0.3.1
  translating ./wayland-egl-backend.h ... ================ V panic ================
   module: builtin
 function: get()
  message: array.get: index out of range (i == 99, a.len == 99)
     file: /home/aakash/.local/bin/v_lang/vlib/builtin/array.v:384
   v hash: ac7e809
=========================================
/tmp/v_1000/../../../../../../home/aakash/.local/bin/v_lang/vlib/builtin/builtin.c.v:53: at panic_debug: Backtrace
/tmp/v_1000/../../../../../../home/aakash/.local/bin/v_lang/vlib/builtin/array.v:384: by array_get
/tmp/v_1000/../../../../../../home/aakash/.vmodules/c2v/c2v.v:788: by main__C2V_record_decl
/tmp/v_1000/../../../../../../home/aakash/.vmodules/c2v/c2v.v:2049: by main__C2V_top_level
/tmp/v_1000/../../../../../../home/aakash/.vmodules/c2v/c2v.v:2004: by main__C2V_translate_file
/tmp/v_1000/../../../../../../home/aakash/.vmodules/c2v/c2v.v:1938: by main__main
/tmp/v_1000/../../../../../../tmp/v_1000/c2v.tmp.c:17582: by main
C2V failed to translate the C files. Please report it via GitHub.
```

Source file: http://0x0.st/o1yR.h

**Comment by ArtemkaKun** [comment]
This is probably fixed in PR #42. I will try to add test today so it can be mergex

**Comment by Shinyzenith** [comment]
Yep the issue has been fixed.
